docs/3.0.0-alpha.x/guides/deployment.md

```diff
@@ -152,9 +152,8 @@ Amazon calls a virtual private server, a **virtual server** or **Amazon EC2 inst
 2. **Select Appropriate Region**. In the top menu, near your IAM Account User name, select, from the dropdown, the most appropriate region to host your Strapi project. For example, `US East (N.Virginia)` or `Asia Pacific (Hong Kong)`. You will want to remember this region for configuring other services on AWS and serving these services from the same region.
 3. Click on the blue `Launch Instance` button.
 
-- \*\*New trial users, click `free tier only` in the left menu
 - `Select` **Ubuntu Server 18.04 LTS (HVM), SSD Volume Type**
-- Ensure `General purpose` + `t2.micro, Free tier eligible` is `checked`.
+- Ensure `General purpose` + `t2.small` is `checked`. **NOTE:** `t2.small` is the smallest instance type in which Strapi runs. `t2.nano` and `t2.micro` **DO NOT** work.
```

Comment:
Add clarification as to why `t2.nano` and `t2.micro` do not work (as I believe this is temporary until @alexandrebodin or @soupette can fix the optimizations on the build process)

Reply:
This was edited in the Beta Documentation to clarify why.